grid2op/tests/test_gymnasium_compat.py

```diff
@@ -5,8 +5,9 @@
 # you can obtain one at http://mozilla.org/MPL/2.0/.
 # SPDX-License-Identifier: MPL-2.0
 # This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
-
-
+import sys
+from pathlib import Path
+sys.path.append(str(Path(__file__).parent))
```

Comment:
It was not necessary before, and tests used to pass on the CI, maybe this is what caused the issue. Can you remove all these changes in the test suite and run it again ?

Reply:
Trying that now :)